proto/minder/v1/minder.proto

```diff
@@ -93,7 +93,10 @@ message Cursor {
     // an per-rpc basis.
     string cursor = 1;
     // size is the number of items to retrieve from the collection.
-    uint32 size = 2;
+    uint32 size = 2 [
+        (buf.validate.field).ignore_empty = true,
```

Comment:
`ignore_empty` seems to be deprecated in favor of:

```
(buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
```

For optional fields, I think you want:

```
(buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
```

Which allows the zero-value as well as "not set at all".

Reply:
Ref: https://buf.build/bufbuild/protovalidate/docs/main:buf.validate#buf.validate.FieldConstraints

Reply:
oh, I'll fix it right away, copilot auto-completion strikes again

Reply:
Ref. https://github.com/mindersec/minder/pull/4882